### DISCUSSION TITLE
Version 0.14.0-beta.3

### DISCUSSION THREAD

**Discussion by trask** [glowroot/glowroot#919] [General]
## What's Changed
* Add transaction type autocomplete on instrumentation page by @ionutursuleanu in https://github.com/glowroot/glowroot/pull/772
* Keep sidebar sort order when changing agent/transaction type by @ionutursuleanu in https://github.com/glowroot/glowroot/pull/726
* Revert trace duration milliseconds title attribute due to low visibility by @trask in https://github.com/glowroot/glowroot/pull/869
* Fix docker image by @trask in https://github.com/glowroot/glowroot/pull/871
* Fix inaccurate free space checking by @ionutursuleanu in https://github.com/glowroot/glowroot/pull/884
* Fix `Copied!` tooltip by @ionutursuleanu in https://github.com/glowroot/glowroot/pull/877
* Issue 773 open shift by @robinroos in https://github.com/glowroot/glowroot/pull/774
* Sidebar top CPU time & allocated memory by @ionutursuleanu in https://github.com/glowroot/glowroot/pull/724
* Optimize docker image size by @Nowheresly in https://github.com/glowroot/glowroot/pull/908
* Use jdk 17 for glowroot central by @Nowheresly in https://github.com/glowroot/glowroot/pull/909

<hr /><em>This discussion was created from the release <a href='https://github.com/glowroot/glowroot/releases/tag/v0.14.0-beta.3'>Version 0.14.0-beta.3</a>.</em>

**Comment by robinroos** [comment]
#907 is not mentioned in the release notes.  Should is also be included in this release?

**Reply by robinroos** [reply]
Ah - I see the effect of #907 is included in #908 which is included in this release.

**Comment by Shinjiru-stack** [comment]
Any update on next stable version @trask ?
